src/bot.js

```diff
@@ -5,6 +5,12 @@ const log4js = require('log4js');
 const client = new Discord.Client();
 var commands = new Map();
 
+// The utils won't be loaded like that because there's no good way to identify them, they also won't be as many
```

Comment:
We should load these dynamically as we do with the commands, probably not something to be done in this branch, but we should at the very least add a TODO comment for it

Reply:
Commented a TODO